cmd/kubeadm/app/master/manifests.go

```diff
@@ -304,7 +304,7 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	}
 
 	command = append(getComponentBaseCommand(apiServer),
-		"--insecure-bind-address=127.0.0.1",
+		"--insecure-port=0",
```

Comment:
TBH -- this is a little confusing.  Generally setting a port to 0 means "randomly select port".  But that isn't a kubeadm issue.